packages/web-app/middleware.ts

```diff
@@ -1,42 +1,45 @@
-import { NextRequest, NextResponse } from 'next/server'
+import { NextRequest, NextResponse } from 'next/server';
```

Comment:
All the changes in this file was just because I ran linter in command line, and they were previously commited by you 🤷‍♂️